ui/cypress/integration/slow_query/01-list.spec.js

```diff
@@ -537,6 +538,102 @@ describe('SlowQuery list page', () => {
     })
   })
 
+  // FIXME: The following tests will break slow-query details E2E since it executes a SQL.
+  // Fix the slow-query details E2E first.
+
+  // describe('Slow network condition', () => {
+  //   const slowNetworkText = 'On-the-fly update is disabled'
+
+  //   it('Does not show slow information when network is fast', () => {
+  //     cy.intercept(`${Cypress.env('apiBasePath')}slow_query/list*`).as(
```

Comment:
These tests are not introduced, since introducing new tests simply breaks the slow query details page E2E. This is horrible. I will not add new E2E tests to slow query until it is fixed.

cc @baurine @YiniXu9506